t/plugin/google-cloud-logging.t

```diff
@@ -751,3 +751,56 @@ GET /hello
 --- wait: 2
 --- response_body
 hello world
+
+
+
+=== TEST 26: set route (verify batch queue default params)
+--- config
+    location /t {
+        content_by_lua_block {
+
+            local config = {
+                uri = "/hello",
+                upstream = {
+                    type = "roundrobin",
+                    nodes = {
+                        ["127.0.0.1:1980"] = 1
+                    }
+                },
+                plugins = {
+                    ["google-cloud-logging"] = {
+                        auth_file = "t/plugin/google-cloud-logging/config.json",
+                    }
+                }
+            }
+
+            local expected = {
+                node = {
+                    value = {
+                        plugins = {
+                            ["google-cloud-logging"] = {
+                                max_retry_count = 0,
+                                retry_delay = 1,
+                                buffer_duration = 60,
+                                batch_max_size = 1000,
+                                inactive_timeout = 5,
+                            }
+                        }
+                    }
+                }
+            }
+
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/routes/1', ngx.HTTP_PUT, config, expected)
+
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(body)
+                return
+            end
+
+            ngx.say(body)
+        }
+    }
+--- response_body
```

Comment:
Let's split it into another test file when it's larger than 800 lines.

Reply:
got, updated